src/token_contract/src/test/main.test.ts

```diff
@@ -27,15 +27,18 @@ import {
 import { EscrowContract, EscrowContractArtifact } from '@aztec/noir-contracts.js/Escrow';
 import { ContractInstanceDeployerContract } from '@aztec/noir-contracts.js/ContractInstanceDeployer';
 
-const startPXE = async (id: number = 0) => {
-  const { PXE_URL = `http://localhost:${8080 + id}` } = process.env;
-  const pxe = createPXEClient(PXE_URL);
+const createPXE = async (id: number = 0) => {
```

Comment:
do we need to change to `createPXE`? what's the criteria for naming?
not against.

Reply:
ah, it uses `createPXEClien` so idk why it was named `start`, i'm ok with it